src/main/java/com/urswolfer/gerrit/client/rest/http/changes/ChangeApiRestClient.java

```diff
@@ -68,6 +66,27 @@ public ChangeApiRestClient(GerritRestClient gerritRestClient,
         this.id = id;
     }
 
+
+    public ChangeApiRestClient(GerritRestClient gerritRestClient,
```

Comment:
Please add the new argument to the default constructor. Otherwise, account parser is not initialized in some cases.